tests/examples/pacer/rss_feeds/ca10.json

```diff
@@ -0,0 +1,188 @@
+[
+  {
+    "assigned_to_str": "",
+    "case_name": "United States v. Kahn",
+    "cause": "",
+    "chapter": "",
+    "court_id": "ca10",
+    "date_converted": null,
+    "date_discharged": null,
+    "date_filed": null,
+    "date_terminated": null,
+    "demand": "",
+    "docket_entries": [
+      {
+        "date_filed": "2023-02-03T09:05:16-07:00",
+        "description": "",
+        "document_number": null,
+        "pacer_doc_id": "010010807970",
```

Comment:
I guess we're normalizing the fourth digit of these, right? I'm not entirely sure that's needed with appellate courts, but we should be sure that we're doing it if we should and not doing it if we shouldn't.

Reply:
Yeah, I've confirmed that we're normalizing the `pacer_doc_id` fourth digit for appellate entries, which is being done in `AppellateDocketReport` and `NotificationEmail`, however, seems that the number is already normalized for appellate entries in the RSS or a docket sheet.